src/OpenTK.Mathematics/Geometry/BezierCurve.cs

```diff
@@ -204,15 +204,15 @@ public static Vector2 CalculatePoint(IList<Vector2> points, float t)
         public static Vector2 CalculatePoint(IList<Vector2> points, float t, float parallel)
         {
             var r = default(Vector2);
-            var c = 1.0d - t;
+            var c = 1 - t;
             float temp;
             var i = 0;
```

Comment:
While we are at it editing this code I think we should take the opportunity to remove some `var`s.

Reply:
I'm personally "team var" but ok, I'll change them in the three I touched here.